reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md

```diff
@@ -140,7 +140,7 @@ await web3AuthModalPack.init(web3AuthModalOptions, adapters, modalConfig)
 // Sign in
 const { eoa, safes } = await web3AuthModalPack.signIn()
 const userInfo = await web3AuthModalPack.getUserInfo()
-const web3Provider = web3AuthModalPack.getProvider()
+const BrowserProvider = web3AuthModalPack.getProvider()
```

Comment:
This can still be web3Provider. It's more clear in this context, it's a parameter name

```suggestion
const web3Provider = web3AuthModalPack.getProvider()
```